charts/veritable-ui/values.yaml

```diff
@@ -107,6 +107,23 @@ idpAuthPath: "/auth"
 idpTokenPath: "/token"
 ## @param idpJWKSPath veritable-ui IDP certs path
 idpJWKSPath: "/certs"
+## @param emailTransport The email transport method to use, current options only STREAM
+emailTransport: "STREAM"
+## @param emailFromAddress veritable-ui email from address
+emailFromAddress: "hello@veritable.com"
+## @param emailAdminAddress veritable-ui email admin address
+emailAdminAddress: "admin@veritable.com"
+## @param cloudagentAdminOrigin veritable-ui cloudagent admin origin URL
+cloudagentAdminOrigin: "http://localhost:3080"
+## @param invitationPin.enabled Enable Invitation pin secret
+## @param invitationPin.secret the secret value
+## @param invitationPin.existingSecret If there is an existing secret for the invitationPin
+## @param invitationPin.existingSecretKey the key to use within the existing secret
+invitationPin:
+  enabled: true
```

Comment:
is the logic here that this should either be enabled or we should pass an existing secret? If so we should add validation logic for that.